src/bin/pg_autoctl/cli_do_tmux.c

```diff
@@ -804,8 +808,10 @@ prepare_tmux_script(TmuxOptions *options, PQExpBuffer script)
 
 		tmux_add_send_keys_command(script,
 								   "PG_AUTOCTL_DEBUG=1 "
+								   "PG_AUTOCTL_DEBUG_BIN_PATH=%s "
```

Comment:
I'm not sure to follow that one. Can't we just use the same `pg_autoctl` path as in `pg_autoctl_argv0`, which should be `src/tools/pg_autoctl.valgrind` at this point right?

Reply:
I removed these specific lines, because it's already set for the whole tmux session now. 

This change in general is needed, because `pg_autoctl_argv0` is the argument that valgrind gets `src/tools/../src/bin/pg_autoctl/pg_autoctl`, not valgrind binary itself. I expect this is the case because valgrind modifies argv, before starting the process.

Reply:
okay, thanks for adding those details, it was quite a mystery to me otherwise!

Reply:
added a comment to the code explaining it too.